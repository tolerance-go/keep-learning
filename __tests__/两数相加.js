const { ListNode, addTwoNumbers } = require("../两数相加");

test("addTwoNumbers", () => {
  const l1_1 = new ListNode(2);
  const l1_2 = new ListNode(4);
  const l1_3 = new ListNode(3);
  l1_1.next = l1_2;
  l1_2.next = l1_3;

  const l2_1 = new ListNode(5);
  const l2_2 = new ListNode(6);
  const l2_3 = new ListNode(4);
  l2_1.next = l2_2;
  l2_2.next = l2_3;

  expect(addTwoNumbers(l1_1, l2_1)).toMatchSnapshot();
});
