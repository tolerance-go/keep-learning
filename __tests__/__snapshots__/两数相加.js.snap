// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addTwoNumbers 1`] = `
ListNode {
  "next": ListNode {
    "next": ListNode {
      "next": null,
      "val": 8,
    },
    "val": 0,
  },
  "val": 7,
}
`;
